ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -2558,6 +2570,7 @@ ECON:
   - outside_air_relative_humidity_sensor
   - outside_air_damper_percentage_sensor
   - return_air_damper_percentage_command
+  - outside_air_damper_percentage_sensor
```

Comment:
@cquigley2 please remove -- looks like it is defined on line 2571 already.

Reply:
removed 2573